<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-7ON0MSH" failures="2" tests="9" name="Test" time="0.275" errors="0" timestamp="2022-08-03T17:44:03 IST">
  <testcase classname="testNgProgram.ItestListenerWithExample" name="failed" time="0.051">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testNgProgram.ItestListenerWithExample.failed(ItestListenerWithExample.java:22)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- failed -->
  <testcase classname="testNgProgram.ItestListenerWithExample" name="testMethod1" time="0.005"/>
  <testcase classname="testNgProgram.ItestListenerWithExample" name="testMethod2" time="0.004">
    <failure type="java.lang.AssertionError" message="expected [11] but found [10]">
      <![CDATA[java.lang.AssertionError: expected [11] but found [10]
at testNgProgram.ItestListenerWithExample.testMethod2(ItestListenerWithExample.java:30)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testMethod2 -->
  <testcase classname="testNgProgram.ItestListenerWithExample" name="testMethod3" time="0.005">
    <skipped/>
  </testcase> <!-- testMethod3 -->
  <testcase classname="testNgProgram.ItestListenerWithExample" name="testMethod4" time="0.006"/>
  <testcase classname="testNgProgram.ItestListenerWithExample" name="testMethod4" time="0.005"/>
  <testcase classname="testNgProgram.ItestListenerWithExample" name="testMethod4" time="0.003"/>
  <testcase classname="testNgProgram.ItestListenerWithExample" name="testMethod4" time="0.003"/>
  <testcase classname="testNgProgram.ItestListenerWithExample" name="testMethod4" time="0.005"/>
</testsuite> <!-- Test -->
